### PR TITLE
Fix eslint errors

### PR DESCRIPTION
**Pull request by Lul-Abdifan**
# Capstone Project 1
## The Files are:
  -  Add Index.html for the home page
  -  Add about.html for the about page
  -  Add script.js for the whole javascript code
  -  Add a style.css file for styling the home page and the about page
  -  Add an assets  folder that contains all the images of the website
  -  Add Linters setup for checking errors in the code
  -  Fix occurred errors
  -  Add gitignore file
  - Fix node module issues
  
## What may differ :
-  Do a mobile menu for the home page
- Do mobile menu for about page
- Add More and Less buttons to add and remove dynamically generated objects
## Github Tasks
-  Deploy the website
- Update the Readme file and make it professional
- Add MIT License
## Short Presentation video
-  [See Demo video](https://www.loom.com/share/b51c3e9eba57445d964476b6d7b3d197)
##### sorry I couldn't change the pull request title once it is created

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Lul-Abdifan, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Invalid Submission

You have submitted a project with node_modules folder not ignored properly. In addition to not being professional, it is difficult to comment on some files.

Also, kindly use a more descriptive title for your pull request that better describes the overall change introduced by the PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Lul-Abdifan** [comment]
@emmiiorji  Thank you for your careful feedback. I appreciate your work .Happy Coding :smile: :+1: :

File: index.html [hunk 1]
```diff
@@ -4,9 +4,6 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
```
Comment (mwanawabangona) [inline index.html:7]: - [ ] `required`
- [ ] Amazing job so far 👏 
- [ ] the text is overlapping please see to it that it is staying the boxes a the origin design
- [ ] also the `join conference button` is not responsive please make reponsive

- [ ] `your template`
image
image



`origin design`
mainspeaker

File: index.html [hunk 2]
```diff
@@ -4,9 +4,6 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  
-
-  
     <!-- google font -->
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```
Comment (mwanawabangona) [inline index.html:9]: - [ ] `required`
- [ ] Also the speaker section is not very responsive you can see from the background color and the overlapping text.
- [ ] Please see screenshots below. 🥂 

`your template`
image

`origin design`
image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @Lul-Abdifan 

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Lul-Abdifan** [comment]
#  Required Chenges are Done
-  Update Partners section more responsive
-   Update Speakers section more responsive
-   Update Join The coming Conference button more responsive
-   Fix overlapping texts
### Note:
-  For speaker since I used many words for position and description It won't be more beautiful unless removing many words. If you have any question. I am here @Lul-Abdifan

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Lul-Abdifan, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

 ## Required Changes ♻️
_Check the comments under the review._
- [ ] Due to insufficient files I will leave a comment here
- [ ] Kindly follow the given design for the `see projects` section
Your design
image

Given design
image

- [ ] The top section does not follow the given template, kindly follow the given template.The section has a `hx` heading, paragraph and  a logo

Given template
image

Your design
image

- [ ] Your top section is missing the `contact` information section, kindly add this.

Your design
image

Given design
image


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Lul-Abdifan** [comment]
## Required Changes are Done
-  Add contact  information section
-  Update about   page totally
-  Add logo for about page

**Review by Gopxfs — APPROVED** [review]
# 🟢 STATUS: APPROVED 🟢

## Hello @Lul-Abdifan 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ You added the contact information on the `about` page.
✔️ Removed the additional picture on the `see projects` section.
✔️ Added a header, a paragraph, and a logo to the about section! Perfect!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Lul-Abdifan** [comment]
@emmiiorji  Thank you for your constructive feedback.:smile::smile:

**Comment by Lul-Abdifan** [comment]
@Gopxfs Thank you for your constructive feedback.:smile::smile:

**Comment by Lul-Abdifan** [comment]
@Gopxfs  Thank you for your constructive feedback.:smile::smile:

**Comment by Lul-Abdifan** [comment]
@mwanawabangona  Thank you for your constructive feedback.:smile::smile:

**Comment by Lul-Abdifan** [comment]
@Munsa1 Thank you for your constructive feedback.:smile::smile:

**Pull request merged**